router/index.js

```diff
@@ -1,47 +1,103 @@
 'use strict';
 
 const express = require('express');
+const db = require('../server/db');
 
 const router = express.Router();
 
+// Home route
+router.get('/', async (req, res, next) => {
+  try {
+    // Get all the sun shade porducts json
+    const sunshades = await db.getSunshades();
+
+    // Send status and render home view
+    res
+      .status(200)
+      .render('home', {
+        pageId: 'home',
+        title: 'Home',
+        sunshades: sunshades.map(sunglasses => ({
+          ...sunglasses,
+        })),
+      });
+  } catch (error) {
+    next(error);
+  }
+});
+
 // Register route
-router.get('/register', (req, res) => {
+router.get('/register', (req, res, next) => {
   res.sendStatus(200);
   // res.send('Register User');
 });
 
 // Login route
-router.get('/login', (req, res) => {
+router.get('/login', (req, res, next) => {
   res.sendStatus(200);
   // res.send('Login User');
 });
 
 // All Product route
-router.get('/product', (req, res) => {
-  // res.send('All Products');
-  res.sendStatus(200);
+router.get('/product', async (req, res, next) => {
+  try {
+    // Get all the sun shade porducts json
```

Comment:
product route successfully returns the json file. Similar to the home route, appropriate status code and try catch properly used.